api/src/main/java/org/openmrs/module/ugandaemr/api/deploy/bundle/CommonMetadataBundle.java

```diff
@@ -69,6 +69,7 @@ public void install() throws Exception {
         install(EncounterTypes.COVID19_ENROLLMENT);
         install(EncounterTypes.COVID19_FOLLOWUP);
         install(EncounterTypes.COVID19_DISCHARGE);
+        install(EncounterTypes.ANC_CARD_ENCOUNTER_TYPE);
```

Comment:
@Mwanje why create a new encounter when we already have these existing in UgandaEMR

Reply:
Done